samples/MvcSample.Web/HomeController.cs

```diff
@@ -113,6 +113,22 @@ public ActionResult SaveUser(User user)
         }
 
         /// <summary>
+        /// Action that shows binding of form values to <see cref="IFormCollection"/>.
+        /// </summary>
+        public ActionResult PostForm(IFormCollection form)
+        {
+            if (!ModelState.IsValid)
```

Comment:
dump the sample completely. We are not interested in advertising this feature. Its there mostly for backcompat (or unit testability for backcompat)